bitnami/redis/Chart.yaml

```diff
@@ -36,4 +36,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 19.1.2
+version: 19.1.3
```

Comment:
I think we should bump the minor version here, as we are adding new features with new parameters.